examples/AN00248_xua_example_pdm_mics/src/xua_conf.h

```diff
@@ -1,31 +1,33 @@
-// Copyright 2017-2022 XMOS LIMITED.
+// Copyright 2017-2024 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 
 #ifndef _XUA_CONF_H_
 #define _XUA_CONF_H_
 
-#define NUM_USB_CHAN_OUT 0
-#define NUM_USB_CHAN_IN 8
-#define I2S_CHANS_DAC 0
-#define I2S_CHANS_ADC 0
-#define MCLK_441 (512 * 44100)
-#define MCLK_48 (512 * 48000)
-#define MIN_FREQ 48000
-#define MAX_FREQ 48000
+#define NUM_USB_CHAN_OUT                    0               /* Number of channels from host to device */
+#define NUM_USB_CHAN_IN                     2               /* Number of channels from device to host */
+#define I2S_CHANS_DAC                       2               /* Number of I2S channels out of xCORE */
+#define I2S_CHANS_ADC                       0               /* Number of I2S channels in to xCORE */
+#define MCLK_441                            (512 * 44100)   /* 44.1kHz family master clock frequency */
+#define MCLK_48                             (512 * 48000)   /* 48kHz family master clock frequency */
+#define XUA_SAMP_FREQ                       48000           /* Currently sample rate changes are not supported for PDM mics */
+#define MIN_FREQ  XUA_SAMP_FREQ                             /* Minimum sample rate */
+#define MAX_FREQ  XUA_SAMP_FREQ                             /* Maximum sample rate */
 
-#define EXCLUDE_USB_AUDIO_MAIN
+#define XUA_NUM_PDM_MICS                    2
+#define MIC_ARRAY_CONFIG_PORT_MCLK          XS1_PORT_1D
```

Comment:
if this file included platform.h the defines generated from the XN file could be used

Reply:
they would be PORT_MCLK_IN, PORT_PDM_CLK and PORT_PDM_DATA

Reply:
Fixed - I thought on the on tile[n] would cause an issue for C/CPP but looks like platform.h does the right thing. Fixed